### PR TITLE
Create Home Page UI

### PR DESCRIPTION
**Pull request by devGyan**

### CODE REVIEW

File: client/src/components/AboutTheProduct/AboutTheProduct.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```
Comment (rhk12) [inline client/src/components/AboutTheProduct/AboutTheProduct.js]: I believe this is the home page? If so it need to replace client/src/pages/HomePage.js.   For now, I would place it there and call it HomePage-2.js

File: client/src/components/AboutTheProduct/AboutTheProductImg/Analysis_White.svg [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="174" height="174" viewBox="0 0 174 174">
+    <defs>
```
Comment (rhk12) [inline client/src/components/AboutTheProduct/AboutTheProductImg/Analysis_White.svg]: I am not completely sure what this file is but if this is image, it should be placed in client/pubilc/img

File: client/src/components/AboutTheProduct/AboutTheProductImg/Cloud_white.svg [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="174" height="174" viewBox="0 0 174 174">
```
Comment (rhk12) [inline client/src/components/AboutTheProduct/AboutTheProductImg/Cloud_white.svg]: same comment as above.

File: client/src/components/AboutTheProduct/AboutTheProductImg/Moniter_white.svg [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="174" height="174" viewBox="0 0 174 174">
+    <defs>
+        <style>
+            .cls-1{fill:#fff}.cls-2{fill:#0f81dc}.cls-3{filter:url(#Ellipse_5)}
+        </style>
```
Comment (rhk12) [inline client/src/components/AboutTheProduct/AboutTheProductImg/Moniter_white.svg]: same comment as above.

File: client/src/components/ResearchArea/ReseachArea.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Soldier from './ResearchAreaImg/Group-2491.svg';
+import Runner from './ResearchAreaImg/Group-2492.svg'
```
Comment (rhk12) [inline client/src/components/ResearchArea/ReseachArea.js]: I am not sure this should be in components. This is part of Home Page. I would put it in client/src/pages/Home/

File: client/src/components/TechnolgiesWeUse/TechnologiesWeUse.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
+import React from 'react';
```
Comment (rhk12) [inline client/src/components/TechnolgiesWeUse/TechnologiesWeUse.js]: Same comment on this file. I feel it should go in the Home directory since it is part of the home page.

File: config/configuration_keys.example.json [hunk 7]
```diff
@@ -1,13 +0,0 @@
-{
```
Comment (eralpsahin) [inline config/configuration_keys.example.json]: Why is this file removed?

Comment (devGyan) [reply config/configuration_keys.example.json]: This file is not removed but is renamed as "configuration_keys.json". because this file is imported as "configuration_keys.json" which is why it was throwing error "module not found".

Comment (eralpsahin) [reply config/configuration_keys.example.json]: I am not seeing a renamed file. It is just removed. [Head of your branch](https://github.com/PSUCompBio/nsfcareer-react-website/tree/createHomeScreenUI/config).

But that does not matter because it should not be removed or renamed. The naming is intentional. This is an example config file, named intentionally as "example". Not designed to be used.

User is supposed to create their own "configurations_keys.json" which is not tracked on GitHub, mentioned [here](https://github.com/PSUCompBio/nsfcareer-react-website#important-do-not-upload-access-keys-to-web-eg-github-google-drive).
